micrometer-core/src/main/java/io/micrometer/core/instrument/binder/cache/CacheMeterBinder.java

```diff
@@ -47,27 +47,23 @@ public CacheMeterBinder(Object cache, String cacheName, Iterable<Tag> tags) {
 
     @Override
     public final void bindTo(MeterRegistry registry) {
-        if (size() != null) {
```

Comment:
Make sense. I've also removed piece of code that defaults to 0 when value is null since we already have null check 
https://github.com/micrometer-metrics/micrometer/blob/master/micrometer-core/src/main/java/io/micrometer/core/instrument/binder/cache/CacheMeterBinder.java#L52-L55

Reply:
I'd like to preserve the null check inside of the gauge as well, for the event that a JCache implementation supports turning stats on and off at runtime. Just extra safety. Want to be really careful that telemetry code never blows up an application.

Reply:
I see. For this purpose I'd rather extract value into local variable prior to null evaluation so that we don't need to do double null check. Other aspect of it - it allows to cover code with unit tests, where with double null check it's close to impossible. Let me know if any questions/concerns.